init.c

```diff
@@ -1184,6 +1184,67 @@ static enum CommandResult parse_group(struct Buffer *buf, struct Buffer *s,
   return MUTT_CMD_WARNING;
 }
 
+/**
+ * find_config_variable - return HashElem of a config_variable
+*/
+static struct HashElem* find_config_variable(const char * name)
+{
+  return cs_get_elem(Config, name);
+}
+
+/**
+ * find_my_variable - return my_variable
+*/
+static const char* find_my_variable(const char * name)
+{
+  return myvar_get(name);
+}
+
+/**
+ * find_environment_variable - return env varible
+*/
+static const char* find_environment_variable(const char * name)
+{
+  return mutt_str_getenv(name);
+}
+
+/**
+ * is_enabled_feature - return bool if argument is an enabled feature
+*/
+static bool is_enabled_feature(const char * name)
+{
+  return feature_enabled(name);
```

Comment:
you can use `feature_enabled(name)` directly in `parse_ifdef` without the need to wrap it